Neos.Ui/core/src/application/LinkTypes/PhoneNumber/PhoneNumber.tsx

```diff
@@ -61,7 +64,11 @@ export const PhoneNumber = makeLinkType<PhoneNumberLinkModel, PhoneNumberLinkOpt
     },
 
     Editor: ({model, options}: {model: Nullable<PhoneNumberLinkModel>, options: OptionalDeep<PhoneNumberLinkOptions>}) => {
+        const [codeArea, setCodeArea] = useState<string>(model?.countryCallingCode || (options?.defaultCountry ? `+${getCountryCallingCode(options?.defaultCountry).toString()}` : `+${getCountryCallingCode(getCountries()[0]).toString()}`));
```

Comment:
The naming is unclear. I think you meant `areaCode`.

Reply:
I changed the variables name